test/app_test.js

```
@@ -5,8 +5,11 @@ const expect = chai.expect;
 const app = require('../src/app');
 const gameState = require('../src/league');
 const leagueRenderer = require('../src/league_renderer');
+const fs = require('fs');
```

Biome

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)